src/main/kotlin/com/chattriggers/ctjs/CTJS.kt

```diff
@@ -87,11 +93,15 @@ object CTJS {
 
         registerHooks()
     }
-
-    fun makeWebRequest(url: String): URLConnection = URL(url).openConnection().apply {
-        setRequestProperty("User-Agent", "Mozilla/5.0 (ChatTriggers)")
-        connectTimeout = 3000
-        readTimeout = 3000
+    fun makeWebRequest(url: String): URLConnection {
+            val connection = URL(url).openConnection()
+            connection.setRequestProperty("User-Agent", "Mozilla/5.0 (ChatTriggers)")
+            if (connection is HttpsURLConnection && sslContext != null) {
+                connection.sslSocketFactory = sslContext!!.socketFactory
```

Comment:
I don't think this null assertion should be necessary? `sslContext` is `val`, so it can't be reassigned

Reply:
It should theoretically not be necessary, but the compiler errors when it's not there